packages/atomic/src/components/atomic-layout/atomic-search-layout.tsx

```diff
@@ -12,8 +12,10 @@ import {buildSearchLayout} from './search-layout';
 })
 export class AtomicSearchLayout {
   @Element() private host!: HTMLElement;
-  // TODO: make breakpoint configurable & reuse in templates
-  private breakpoint = '1024px';
+  /**
+   * CSS value, in px, that defines where the layout goes from mobile to desktop.
+   */
+  @Prop() public breakpoint = 1024;
```

Comment:
I prefer giving people the option to use the unit they want by passing a string (in case they use `em`/`rem`, for instance).

Reply:
Is a single breakpoint enough 🤔 

Should we leave room to expand on it in the future for layouts other than search ?

So something like an array, or a complex object might be more appropriate ?

Reply:
since it's now a specific atomic-search-layout component, it won't be a problem to add more for other layouts, I've renamed it to mobile-breakpoint to be sure